prow/cluster/plank_deployment.yaml

```diff
@@ -29,13 +29,22 @@ spec:
         role: prow
       containers:
       - name: plank
-        image: gcr.io/k8s-prow/plank:0.17
+        image: gcr.io/k8s-prow/plank:0.18
+        args:
+        - "--dry-run=false"
+        - "--github-bot-name=k8s-ci-robot"
```

Comment:
I would love to submit a follow-up PR and make the move if you guys are ok with it. I am more inclined to put it in the config but tying it with the oauth token in the secret is not unreasonable too. 

I have also found other instances of things that should flow from the prow config as opposed to hardcoding them[1][2] that I would like to make configurable.

[1] https://github.com/kubernetes/test-infra/blob/99d3864c96daef27bead37cc1b9dce63f799f587/prow/plugins/trigger/pr.go#L79 (bot name)
[2] https://github.com/kubernetes/test-infra/blob/99d3864c96daef27bead37cc1b9dce63f799f587/prow/plugins/trigger/trigger.go#L31 (org name)